modules/channel-smooch/README.md

```diff
@@ -0,0 +1,37 @@
+# Channel-Smooch
```

Comment:
I forgot that we don't add the instructions in the README file anymore. You can move the content of your readme file to the documentation page, and add a link to the documentation in this readme file.

Here: https://github.com/botpress/botpress/tree/master/docs/guide/docs/channels